modules/Clinical/domains/vital_status.yaml

```diff
@@ -41,28 +29,44 @@ enums:
       "Unknown":
         description: Not known, not observed, not recorded, or refused
 
+  VitalStatusEnum:
+    title: vital_status
+    permissible_values:
+      "Alive":
+        description: Alive
+      "Deceased":
+        description: Deceased
+      "Not Reported":
+        description: Not provided or available
+      "Unknown":
+        description: Not known, not observed, not recorded, or refused
+
 classes:
   VitalStatus:
     description: Information about the vital status
     attributes:
       VITAL_STATUS:
         range: VitalStatusEnum
         required: true
+        title: "Vital Status"
```

Comment:
We should add to our design doc or enum a style guide (if we don't have one already.

attribute slots are "ALL_CAPS_WITH_UNDERSCORES"
enums are "CamelCase"
titles are "Title Case"
descriptions are "First word capital letter"